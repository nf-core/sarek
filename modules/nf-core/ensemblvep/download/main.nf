process ENSEMBLVEP_DOWNLOAD {
    tag "$meta.id"
    label 'process_medium'

    conda "${moduleDir}/environment.yml"
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/ensembl-vep:113.0--pl5321h2a3209d_0' :
        'biocontainers/ensembl-vep:113.0--pl5321h2a3209d_0' }"

    input:
    tuple val(meta), val(assembly), val(species), val(cache_version)

    output:
    tuple val(meta), path(prefix), emit: cache
    path "versions.yml"          , emit: versions

    when:
    task.ext.when == null || task.ext.when

    script:
    def args = task.ext.args ?: ''
    prefix = task.ext.prefix ?: 'vep_cache'
    """
    vep_install \\
        --CACHEDIR $prefix \\
        --SPECIES $species \\
        --ASSEMBLY $assembly \\
        --CACHE_VERSION $cache_version \\
        $args

    cat <<-END_VERSIONS > versions.yml
    "${task.process}":
        ensemblvep: \$( echo \$(vep --help 2>&1) | sed 's/^.*Versions:.*ensembl-vep : //;s/ .*\$//')
    END_VERSIONS
    """

    stub:
    prefix = task.ext.prefix ?: 'vep_cache'
    """
    mkdir $prefix

    cat <<-END_VERSIONS > versions.yml
    "${task.process}":
        ensemblvep: \$( echo \$(vep --help 2>&1) | sed 's/^.*Versions:.*ensembl-vep : //;s/ .*\$//')
    END_VERSIONS
    """
}
