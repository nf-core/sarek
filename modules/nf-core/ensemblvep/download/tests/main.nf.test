nextflow_process {

    name "Test Process ENSEMBLVEP_DOWNLOAD"
    script "../main.nf"
    process "ENSEMBLVEP_DOWNLOAD"
    config "./nextflow.config"

    tag "modules"
    tag "modules_nfcore"
    tag "ensemblvep"
    tag "ensemblvep/download"

    test("celegans - download") {

        when {
            process {
                """
                input[0] = Channel.of([
                    [id:"113_WBcel235"],
                    params.vep_genome,
                    params.vep_species,
                    params.vep_cache_version
                ])
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }
    }

    test("celegans - download - stub") {

        options "-stub"

        when {
            process {
                """
                input[0] = Channel.of([
                    [id:"113_WBcel235"],
                    params.vep_genome,
                    params.vep_species,
                    params.vep_cache_version
                ])
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }
    }
}
