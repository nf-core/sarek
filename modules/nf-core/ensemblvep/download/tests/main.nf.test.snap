{
    "celegans - download": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "115_WBcel235"
                        },
                        [
                            [
                                [
                                    [
                                        "1-1000000.gz:md5,bb3f43b7512715df72792988ed262d0e",
                                        "10000001-11000000.gz:md5,3a2acf075f478e42dab768cc3913d3f7",
                                        "1000001-2000000.gz:md5,2a1fa1b46a47f90fe36b5c8ab35d01a1",
                                        "11000001-12000000.gz:md5,e16032d944e54a4f41a82925817f88c6",
                                        "12000001-13000000.gz:md5,f51d695b264752299a031b9dbcc44de1",
                                        "13000001-14000000.gz:md5,1532d1764ee39bfd8e1be8b50974ab29",
                                        "14000001-15000000.gz:md5,24173fdc2f65e39aac7ecc4c2726fcf2",
                                        "15000001-16000000.gz:md5,d36d8154114de70c6779ebfb08f1ff8a",
                                        "2000001-3000000.gz:md5,dadb500858b247daf0cf5562938a081d",
                                        "3000001-4000000.gz:md5,eaaebe12f3402b7794c1d82a677ff5ad",
                                        "4000001-5000000.gz:md5,84633b5ee211f4e23170233aefc415ee",
                                        "5000001-6000000.gz:md5,de297b2d604da2070f9630a83db73b22",
                                        "6000001-7000000.gz:md5,0fa47dad82ca5bce6ef96afc491a602c",
                                        "7000001-8000000.gz:md5,2387061609a13eeaeb89a90dd13b1495",
                                        "8000001-9000000.gz:md5,1bfd2fea5051d0a09dde436e75733739",
                                        "9000001-10000000.gz:md5,dbb48f71832f61b9ec01c6f60548e954"
                                    ],
                                    [
                                        "1-1000000.gz:md5,701d7a2d7afa18b4efd7db488089f162",
                                        "10000001-11000000.gz:md5,105bf3809d7be6f5c310383d5013e633",
                                        "1000001-2000000.gz:md5,9b9eae5303b82666138619afd7b562dd",
                                        "11000001-12000000.gz:md5,c5cc2c2dd5afc3028f3f0a7c03c067c1",
                                        "12000001-13000000.gz:md5,721a1108f6d3b83f1ec96cf39ac9844d",
                                        "13000001-14000000.gz:md5,9fec39204bcdcc5e0458b4ae19ca6f60",
                                        "14000001-15000000.gz:md5,234b0a0b377b6740b70b29ecf0beb9fb",
                                        "15000001-16000000.gz:md5,18444392a5aa3a7e2f09606f3481b765",
                                        "2000001-3000000.gz:md5,e7ed6a622c2759de225b4602feabc1d1",
                                        "3000001-4000000.gz:md5,fde2094cb236ed5cca8f1d6ef2d1e2ed",
                                        "4000001-5000000.gz:md5,2035077fec25cb75beaaf61a0801ec0b",
                                        "5000001-6000000.gz:md5,7d5534332f9aee366ad57add71bb0625",
                                        "6000001-7000000.gz:md5,3da3614a656ed04dd8ba6f06b4c12c37",
                                        "7000001-8000000.gz:md5,f24538b6263452002c9eef88657f332a",
                                        "8000001-9000000.gz:md5,c1357499904580cb780ef275493d06a5",
                                        "9000001-10000000.gz:md5,ed7fee5ec049da066556e033cec1f9b1"
                                    ],
                                    [
                                        "1-1000000.gz:md5,e4167c4f03a10c72e22231bce4138383",
                                        "10000001-11000000.gz:md5,a0d9708cbad465edc12f3ce420aa2516",
                                        "1000001-2000000.gz:md5,0b7849c9659b74518f94f0ff760c07c3",
                                        "11000001-12000000.gz:md5,743d76706105cf702cd1eac015c0afd3",
                                        "12000001-13000000.gz:md5,f66f0aa9a02ab3e781cb04d10cc05093",
                                        "13000001-14000000.gz:md5,56dd08b3ddc0b003221e1a1d16331d61",
                                        "2000001-3000000.gz:md5,5477e38b3d53108e4e8ef70b4576dacd",
                                        "3000001-4000000.gz:md5,bd40e0f8602020d14e3e38aeb3a9f31c",
                                        "4000001-5000000.gz:md5,6ae06d70e1fa445e46d66e69b0ca629f",
                                        "5000001-6000000.gz:md5,db03cd98e32f6adb5b424a81420b983c",
                                        "6000001-7000000.gz:md5,4dffec87c3e18cd5c5cdb0befa3230c8",
                                        "7000001-8000000.gz:md5,96ed0ed8f00382c43f6686f70f2b3925",
                                        "8000001-9000000.gz:md5,5c20ed1c637e1182766e66f5741bd070",
                                        "9000001-10000000.gz:md5,2ecd0f2778a09425a2c83e4b5e76ba1b"
                                    ],
                                    [
                                        "1-1000000.gz:md5,28ab3ff34fdd100a65afb89b4ae0f825",
                                        "10000001-11000000.gz:md5,feaf25015c12e72ea5c9e444ffb9a6a9",
                                        "1000001-2000000.gz:md5,8878c44d092a2f7de7298bcecc43da73",
                                        "11000001-12000000.gz:md5,60764928b738a5afb38e739747945ee0",
                                        "12000001-13000000.gz:md5,a397368a23323df7104469537ef4e363",
                                        "13000001-14000000.gz:md5,ecefb54ca22247017c75d8af2fc72e67",
                                        "14000001-15000000.gz:md5,513213c222473a434e73b79927fa3eb3",
                                        "15000001-16000000.gz:md5,b381b0e58227aec9cd1e78f3c2ace485",
                                        "16000001-17000000.gz:md5,44a78638fbdd9d56f20462ba50867d14",
                                        "17000001-18000000.gz:md5,fb6bdfe6078d861895a83f4f7232f74d",
                                        "2000001-3000000.gz:md5,3fe93a5ff45260e3814e177e0f91a9e9",
                                        "3000001-4000000.gz:md5,d449609384c8c0a6cf159b556ef29d47",
                                        "4000001-5000000.gz:md5,201ec9eb75148838ef97bb931acdc02f",
                                        "5000001-6000000.gz:md5,44166dce1082f3e3ce9128930a7aa6a3",
                                        "6000001-7000000.gz:md5,0a54b8acee2e4531423079f361e465ee",
                                        "7000001-8000000.gz:md5,17f245e52522d1270e7005a633eea682",
                                        "8000001-9000000.gz:md5,5f364dabdef97a204e89bce1b9835822",
                                        "9000001-10000000.gz:md5,bb77aff8a7673b2153476b0b9374ffd8"
                                    ],
                                    [
                                        "1-1000000.gz:md5,ad7333dd27732b000f1fb91c69accc47"
                                    ],
                                    [
                                        "1-1000000.gz:md5,a104f41c6b74d3657094341960b3e9aa",
                                        "10000001-11000000.gz:md5,cb73357d42446a2a62d41b4ab6814486",
                                        "1000001-2000000.gz:md5,c3d1f590019cb2d9a3fe138ebc3d4adb",
                                        "11000001-12000000.gz:md5,f5de9fb9e6a24c3a0176951fe70cfd35",
                                        "12000001-13000000.gz:md5,ad46cdbd9087eeac328e8dcecb60bdd6",
                                        "13000001-14000000.gz:md5,635dbb498a367214ca45ea4466e15c27",
                                        "14000001-15000000.gz:md5,b4e72a2a01c2d524c1076a9aeb5be6ed",
                                        "15000001-16000000.gz:md5,4ce9dd41ed032eac6234ae6d414dead8",
                                        "16000001-17000000.gz:md5,908fc89908bf5be9862c9f989e105a97",
                                        "17000001-18000000.gz:md5,7bda940567e9c6fa9ba5493911e607b7",
                                        "18000001-19000000.gz:md5,94c81ccdf2110781989774198c725e76",
                                        "19000001-20000000.gz:md5,8e39cf5f5b24bb28c117d502a5d4cd64",
                                        "20000001-21000000.gz:md5,d9a72ba97be5648b08ac53bc4649086e",
                                        "2000001-3000000.gz:md5,e90cc7adc6c60b2b37736f572fe79166",
                                        "3000001-4000000.gz:md5,6b5bbecf824fc18ce9b46c1cbedf0984",
                                        "4000001-5000000.gz:md5,beccbf5aee5031d2d6bc29f2c7486672",
                                        "5000001-6000000.gz:md5,23810c84b4f8607537321f0823a48db3",
                                        "6000001-7000000.gz:md5,d9008100f5e23d3f42849d2bb9c45432",
                                        "7000001-8000000.gz:md5,079f7fa854d40d426925d85a58d688e0",
                                        "8000001-9000000.gz:md5,f000a7feaa7b461cdb6fc3df3651f6ae",
                                        "9000001-10000000.gz:md5,154d5d7cf4d70cc3d9ea2349af7103be"
                                    ],
                                    [
                                        "1-1000000.gz:md5,d7a153a75f81d0b45eab68bfe526ffb5",
                                        "10000001-11000000.gz:md5,db38af65cff53b1d1f06489cc872b015",
                                        "1000001-2000000.gz:md5,0a7fff7f67d786d9281ca4783a5ec163",
                                        "11000001-12000000.gz:md5,d3231f8fd2e05281d4388d67c5dfaed2",
                                        "12000001-13000000.gz:md5,779879ffc3cdf8cd4021fef90b130791",
                                        "13000001-14000000.gz:md5,0dea4526b0aa9085afae63249ab361ba",
                                        "14000001-15000000.gz:md5,c25184561e98dd04343f49a8dc339134",
                                        "15000001-16000000.gz:md5,146861be6570e64a0ceb555c9f8e3367",
                                        "16000001-17000000.gz:md5,8e497ba1fe43f39adc47f625b2f0d0cd",
                                        "17000001-18000000.gz:md5,d90fbba28b69487d8948e22f1b7b4217",
                                        "2000001-3000000.gz:md5,2c6990365afec01242bcf698edc7e9da",
                                        "3000001-4000000.gz:md5,9b6c90fd96f4d781d1ccf1339c7fb094",
                                        "4000001-5000000.gz:md5,ee14bc5bfb88da22b27004a948570904",
                                        "5000001-6000000.gz:md5,b8f609f3fb3d607f2da254a217b9fc6a",
                                        "6000001-7000000.gz:md5,be9317f3c10182edd13426cdab8e1607",
                                        "7000001-8000000.gz:md5,a5ab699b93ad721e676f0694a288ae36",
                                        "8000001-9000000.gz:md5,6b5e1b10f154d5b046b7ac58f0957fcb",
                                        "9000001-10000000.gz:md5,db16ac0616d71ce8647e885be9ed0cad"
                                    ],
                                    "chr_synonyms.txt:md5,9b3745f472606bf05c7068b21e2b31aa",
                                    "info.txt:md5,33ccb74a030a9a345051628c337cb8af"
                                ]
                            ]
                        ]
                    ]
                ],
                "1": [
                    "versions.yml:md5,906ef8bda2ae1461f20649a6ba8611f1"
                ],
                "cache": [
                    [
                        {
                            "id": "115_WBcel235"
                        },
                        [
                            [
                                [
                                    [
                                        "1-1000000.gz:md5,bb3f43b7512715df72792988ed262d0e",
                                        "10000001-11000000.gz:md5,3a2acf075f478e42dab768cc3913d3f7",
                                        "1000001-2000000.gz:md5,2a1fa1b46a47f90fe36b5c8ab35d01a1",
                                        "11000001-12000000.gz:md5,e16032d944e54a4f41a82925817f88c6",
                                        "12000001-13000000.gz:md5,f51d695b264752299a031b9dbcc44de1",
                                        "13000001-14000000.gz:md5,1532d1764ee39bfd8e1be8b50974ab29",
                                        "14000001-15000000.gz:md5,24173fdc2f65e39aac7ecc4c2726fcf2",
                                        "15000001-16000000.gz:md5,d36d8154114de70c6779ebfb08f1ff8a",
                                        "2000001-3000000.gz:md5,dadb500858b247daf0cf5562938a081d",
                                        "3000001-4000000.gz:md5,eaaebe12f3402b7794c1d82a677ff5ad",
                                        "4000001-5000000.gz:md5,84633b5ee211f4e23170233aefc415ee",
                                        "5000001-6000000.gz:md5,de297b2d604da2070f9630a83db73b22",
                                        "6000001-7000000.gz:md5,0fa47dad82ca5bce6ef96afc491a602c",
                                        "7000001-8000000.gz:md5,2387061609a13eeaeb89a90dd13b1495",
                                        "8000001-9000000.gz:md5,1bfd2fea5051d0a09dde436e75733739",
                                        "9000001-10000000.gz:md5,dbb48f71832f61b9ec01c6f60548e954"
                                    ],
                                    [
                                        "1-1000000.gz:md5,701d7a2d7afa18b4efd7db488089f162",
                                        "10000001-11000000.gz:md5,105bf3809d7be6f5c310383d5013e633",
                                        "1000001-2000000.gz:md5,9b9eae5303b82666138619afd7b562dd",
                                        "11000001-12000000.gz:md5,c5cc2c2dd5afc3028f3f0a7c03c067c1",
                                        "12000001-13000000.gz:md5,721a1108f6d3b83f1ec96cf39ac9844d",
                                        "13000001-14000000.gz:md5,9fec39204bcdcc5e0458b4ae19ca6f60",
                                        "14000001-15000000.gz:md5,234b0a0b377b6740b70b29ecf0beb9fb",
                                        "15000001-16000000.gz:md5,18444392a5aa3a7e2f09606f3481b765",
                                        "2000001-3000000.gz:md5,e7ed6a622c2759de225b4602feabc1d1",
                                        "3000001-4000000.gz:md5,fde2094cb236ed5cca8f1d6ef2d1e2ed",
                                        "4000001-5000000.gz:md5,2035077fec25cb75beaaf61a0801ec0b",
                                        "5000001-6000000.gz:md5,7d5534332f9aee366ad57add71bb0625",
                                        "6000001-7000000.gz:md5,3da3614a656ed04dd8ba6f06b4c12c37",
                                        "7000001-8000000.gz:md5,f24538b6263452002c9eef88657f332a",
                                        "8000001-9000000.gz:md5,c1357499904580cb780ef275493d06a5",
                                        "9000001-10000000.gz:md5,ed7fee5ec049da066556e033cec1f9b1"
                                    ],
                                    [
                                        "1-1000000.gz:md5,e4167c4f03a10c72e22231bce4138383",
                                        "10000001-11000000.gz:md5,a0d9708cbad465edc12f3ce420aa2516",
                                        "1000001-2000000.gz:md5,0b7849c9659b74518f94f0ff760c07c3",
                                        "11000001-12000000.gz:md5,743d76706105cf702cd1eac015c0afd3",
                                        "12000001-13000000.gz:md5,f66f0aa9a02ab3e781cb04d10cc05093",
                                        "13000001-14000000.gz:md5,56dd08b3ddc0b003221e1a1d16331d61",
                                        "2000001-3000000.gz:md5,5477e38b3d53108e4e8ef70b4576dacd",
                                        "3000001-4000000.gz:md5,bd40e0f8602020d14e3e38aeb3a9f31c",
                                        "4000001-5000000.gz:md5,6ae06d70e1fa445e46d66e69b0ca629f",
                                        "5000001-6000000.gz:md5,db03cd98e32f6adb5b424a81420b983c",
                                        "6000001-7000000.gz:md5,4dffec87c3e18cd5c5cdb0befa3230c8",
                                        "7000001-8000000.gz:md5,96ed0ed8f00382c43f6686f70f2b3925",
                                        "8000001-9000000.gz:md5,5c20ed1c637e1182766e66f5741bd070",
                                        "9000001-10000000.gz:md5,2ecd0f2778a09425a2c83e4b5e76ba1b"
                                    ],
                                    [
                                        "1-1000000.gz:md5,28ab3ff34fdd100a65afb89b4ae0f825",
                                        "10000001-11000000.gz:md5,feaf25015c12e72ea5c9e444ffb9a6a9",
                                        "1000001-2000000.gz:md5,8878c44d092a2f7de7298bcecc43da73",
                                        "11000001-12000000.gz:md5,60764928b738a5afb38e739747945ee0",
                                        "12000001-13000000.gz:md5,a397368a23323df7104469537ef4e363",
                                        "13000001-14000000.gz:md5,ecefb54ca22247017c75d8af2fc72e67",
                                        "14000001-15000000.gz:md5,513213c222473a434e73b79927fa3eb3",
                                        "15000001-16000000.gz:md5,b381b0e58227aec9cd1e78f3c2ace485",
                                        "16000001-17000000.gz:md5,44a78638fbdd9d56f20462ba50867d14",
                                        "17000001-18000000.gz:md5,fb6bdfe6078d861895a83f4f7232f74d",
                                        "2000001-3000000.gz:md5,3fe93a5ff45260e3814e177e0f91a9e9",
                                        "3000001-4000000.gz:md5,d449609384c8c0a6cf159b556ef29d47",
                                        "4000001-5000000.gz:md5,201ec9eb75148838ef97bb931acdc02f",
                                        "5000001-6000000.gz:md5,44166dce1082f3e3ce9128930a7aa6a3",
                                        "6000001-7000000.gz:md5,0a54b8acee2e4531423079f361e465ee",
                                        "7000001-8000000.gz:md5,17f245e52522d1270e7005a633eea682",
                                        "8000001-9000000.gz:md5,5f364dabdef97a204e89bce1b9835822",
                                        "9000001-10000000.gz:md5,bb77aff8a7673b2153476b0b9374ffd8"
                                    ],
                                    [
                                        "1-1000000.gz:md5,ad7333dd27732b000f1fb91c69accc47"
                                    ],
                                    [
                                        "1-1000000.gz:md5,a104f41c6b74d3657094341960b3e9aa",
                                        "10000001-11000000.gz:md5,cb73357d42446a2a62d41b4ab6814486",
                                        "1000001-2000000.gz:md5,c3d1f590019cb2d9a3fe138ebc3d4adb",
                                        "11000001-12000000.gz:md5,f5de9fb9e6a24c3a0176951fe70cfd35",
                                        "12000001-13000000.gz:md5,ad46cdbd9087eeac328e8dcecb60bdd6",
                                        "13000001-14000000.gz:md5,635dbb498a367214ca45ea4466e15c27",
                                        "14000001-15000000.gz:md5,b4e72a2a01c2d524c1076a9aeb5be6ed",
                                        "15000001-16000000.gz:md5,4ce9dd41ed032eac6234ae6d414dead8",
                                        "16000001-17000000.gz:md5,908fc89908bf5be9862c9f989e105a97",
                                        "17000001-18000000.gz:md5,7bda940567e9c6fa9ba5493911e607b7",
                                        "18000001-19000000.gz:md5,94c81ccdf2110781989774198c725e76",
                                        "19000001-20000000.gz:md5,8e39cf5f5b24bb28c117d502a5d4cd64",
                                        "20000001-21000000.gz:md5,d9a72ba97be5648b08ac53bc4649086e",
                                        "2000001-3000000.gz:md5,e90cc7adc6c60b2b37736f572fe79166",
                                        "3000001-4000000.gz:md5,6b5bbecf824fc18ce9b46c1cbedf0984",
                                        "4000001-5000000.gz:md5,beccbf5aee5031d2d6bc29f2c7486672",
                                        "5000001-6000000.gz:md5,23810c84b4f8607537321f0823a48db3",
                                        "6000001-7000000.gz:md5,d9008100f5e23d3f42849d2bb9c45432",
                                        "7000001-8000000.gz:md5,079f7fa854d40d426925d85a58d688e0",
                                        "8000001-9000000.gz:md5,f000a7feaa7b461cdb6fc3df3651f6ae",
                                        "9000001-10000000.gz:md5,154d5d7cf4d70cc3d9ea2349af7103be"
                                    ],
                                    [
                                        "1-1000000.gz:md5,d7a153a75f81d0b45eab68bfe526ffb5",
                                        "10000001-11000000.gz:md5,db38af65cff53b1d1f06489cc872b015",
                                        "1000001-2000000.gz:md5,0a7fff7f67d786d9281ca4783a5ec163",
                                        "11000001-12000000.gz:md5,d3231f8fd2e05281d4388d67c5dfaed2",
                                        "12000001-13000000.gz:md5,779879ffc3cdf8cd4021fef90b130791",
                                        "13000001-14000000.gz:md5,0dea4526b0aa9085afae63249ab361ba",
                                        "14000001-15000000.gz:md5,c25184561e98dd04343f49a8dc339134",
                                        "15000001-16000000.gz:md5,146861be6570e64a0ceb555c9f8e3367",
                                        "16000001-17000000.gz:md5,8e497ba1fe43f39adc47f625b2f0d0cd",
                                        "17000001-18000000.gz:md5,d90fbba28b69487d8948e22f1b7b4217",
                                        "2000001-3000000.gz:md5,2c6990365afec01242bcf698edc7e9da",
                                        "3000001-4000000.gz:md5,9b6c90fd96f4d781d1ccf1339c7fb094",
                                        "4000001-5000000.gz:md5,ee14bc5bfb88da22b27004a948570904",
                                        "5000001-6000000.gz:md5,b8f609f3fb3d607f2da254a217b9fc6a",
                                        "6000001-7000000.gz:md5,be9317f3c10182edd13426cdab8e1607",
                                        "7000001-8000000.gz:md5,a5ab699b93ad721e676f0694a288ae36",
                                        "8000001-9000000.gz:md5,6b5e1b10f154d5b046b7ac58f0957fcb",
                                        "9000001-10000000.gz:md5,db16ac0616d71ce8647e885be9ed0cad"
                                    ],
                                    "chr_synonyms.txt:md5,9b3745f472606bf05c7068b21e2b31aa",
                                    "info.txt:md5,33ccb74a030a9a345051628c337cb8af"
                                ]
                            ]
                        ]
                    ]
                ],
                "versions": [
                    "versions.yml:md5,906ef8bda2ae1461f20649a6ba8611f1"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-09-04T10:00:46.010443"
    },
    "celegans - download - stub": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "115_WBcel235"
                        },
                        [
                            
                        ]
                    ]
                ],
                "1": [
                    "versions.yml:md5,906ef8bda2ae1461f20649a6ba8611f1"
                ],
                "cache": [
                    [
                        {
                            "id": "115_WBcel235"
                        },
                        [
                            
                        ]
                    ]
                ],
                "versions": [
                    "versions.yml:md5,906ef8bda2ae1461f20649a6ba8611f1"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-09-04T10:00:57.964733"
    }
}